src/plugins/visibilityCleaner.ts

```diff
@@ -52,6 +52,16 @@ class VisibilityCleaner {
                 }
             }
         }
+
+        if (visible === CliCommonSchema.CodeModel.Visibility.false) {
+            if (NodeHelper.HasSubClass(schema)) {
+                for (let subClass of NodeHelper.getSubClasses(schema, true)) {
+                    if (this.calcObject(subClass) === CliCommonSchema.CodeModel.Visibility.true)
+                        visible = CliCommonSchema.CodeModel.Visibility.true;
```

Comment:
I think as long as one subclass is true, `visible` is true. So can we add break once we find one visible subclass?

Reply:
good point. updated. thx